### PR TITLE
Bug 1617924 - Samples can't be tried and downloaded

### PR DESCRIPTION
**Pull request by jhadvig**
Samples for NetworkPolicy couldn't be tested.
Also samples for all the resources with sidebar in the YamlEditor(BC, NetworkPolicy, Roles) couldnt not be downloaded.

@spadgett PTAL

### CODE REVIEW

File: frontend/public/components/sidebars/network-policy-sidebar.jsx [hunk 1]
```diff
@@ -16,54 +18,61 @@ const samples = [
     img: denyOtherNamespacesImg,
     details: 'Deny traffic from other namespaces while allowing all traffic from the namespaces the Pod is living in.',
     templateName: 'deny-other-namespaces',
+    kind: referenceForModel(NetworkPolicyModel),
```
Comment (alecmerdler) [inline frontend/public/components/sidebars/network-policy-sidebar.jsx]: I swear I added these...

Comment (jhadvig) [reply frontend/public/components/sidebars/network-policy-sidebar.jsx]: NP at all @alecmerdler !!! the more we fix the more we learn ! :)

Comment (jhadvig) [reply frontend/public/components/sidebars/network-policy-sidebar.jsx]: Actually I really like how the YAML editor with its bits and pieces works :+1:

Comment (alecmerdler) [reply frontend/public/components/sidebars/network-policy-sidebar.jsx]: Have you checked out the YAML editor autocompletion?

**Comment by spadgett** [comment]
> Filtering : filters Pod from object detail

This flake is happening a lot :(

**Comment by spadgett** [comment]
jenkins rebuild

**Comment by spadgett** [comment]
jenkins rebuild

**Pull request merged**